### PR TITLE
Nightlies

### PR DESCRIPTION
**Pull request by himadrisingh**

### CODE REVIEW

File: .github/workflows/cli-release.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ on:
   push:
     tags:
       - "**"
+  schedule:
+    - cron: "0,10,20,30,40,50 * * * *"
```
Comment (begelundmuller) [inline .github/workflows/cli-release.yml:8]: Update to daily before merging

**Review by begelundmuller — APPROVED** [review]
Looks good!

File: .github/workflows/cli-release.yml [hunk 2]
```diff
@@ -4,6 +4,8 @@ on:
   push:
     tags:
       - "**"
+  schedule:
+    - cron: '0,10,20,30,40,50 * * * *'
```
Comment (himadrisingh) [inline .github/workflows/cli-release.yml:8]:
```suggestion
  schedule:
    - cron: '0 * * * *'
```

File: .goreleaser.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ project_name: rill
 
 release:
   prerelease: auto
-  draft: false
+  draft: true
```
Comment (himadrisingh) [inline .goreleaser.yaml:5]:
```suggestion
  draft: false
```

File: .github/workflows/cli-release.yml (hunk 2 above)
Comment (himadrisingh) [reply .github/workflows/cli-release.yml:8]: running it every hour for now to test

**Pull request merged**